Apps/ValidationTests/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+set(SCRIPTS "Scripts/validation_native.js")
+set(BABYLONJS 
+    "../Babylonjs/babylon.glTF2FileLoader.js"
+    "../Babylonjs/babylon.max.js"
+    "../Babylonjs/babylonjs.materials.js")
+
+set(SOURCES
+    ${SCRIPTS}
+    ${BABYLONJS}
+    "Shared/TestUtils.h")
+if(WIN32 AND NOT WINDOWS_STORE)
+    set(SOURCES
+        ${SOURCES}
+        "Win32/App.cpp"
+        "Win32/App.h"
+        "Win32/App.ico"
+        "Win32/App.rc"
+        "Win32/Resource.h"
+        "Win32/small.ico"
+        "Win32/targetver.h")
+endif()
+
+if(WIN32)
+    set(WIN32_EXECUTABLE TRUE)
+    add_executable(ValidationTests WIN32 ${SOURCES} ${RESOURCE_FILES})
+endif()
```

Comment:
Should we throw out a readable message here indicating that validation tests aren't supported on a given platform? Just to make it clear what went wrong in case someone tries.

Reply:
I'd rather put the info in the doc. Not sure anyone reads the cmake output when there is no error.

Reply:
I meant have an else case here to throw a readable error message if somebody tries to build this project on a non-Windows platform.